Formula/texlive.rb

```diff
@@ -0,0 +1,165 @@
+class Texlive < Formula
+  desc "Free software distribution for the TeX typesetting system"
+  homepage "https://www.tug.org/texlive/"
+  url "https://github.com/TeX-Live/texlive-source/archive/refs/tags/svn58837.tar.gz"
+  sha256 "0afa6919e44675b7afe0fa45344747afef07b6ee98eeb14ff6a2ef78f458fc12"
+  license :public_domain
+
+  depends_on "cairo"
+  depends_on "clisp"
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "gd"
+  depends_on "ghostscript"
+  depends_on "gmp"
+  depends_on "graphite2"
+  depends_on "harfbuzz"
+  depends_on "libpng"
+  depends_on "lua"
+  depends_on "luajit-openresty"
+  depends_on "mpfr"
+  depends_on "perl"
+  depends_on "pixman"
+  depends_on "potrace"
+
+  uses_from_macos "icu4c"
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "gcc"
+    depends_on "libice"
+    depends_on "libsm"
+    depends_on "libx11"
+    depends_on "libxaw"
+    depends_on "libxext"
+    depends_on "libxmu"
+    depends_on "libxpm"
+    depends_on "libxt"
+  end
+
+  fails_with gcc: "5"
+
+  resource("texlive-extra") do
+    url "ftp://tug.org/texlive/historic/2021/texlive-20210325-extra.tar.xz"
```

Comment:
It seems that there is some problem with the FTP protocol in CI. Would it be the case to use an HTTPS mirror?

Reply:
Oddly this only failed on 10.15.  Linux and the other macOS versions ran into other problems but downloaded the source with no issues.